app/views/clubs/index.html.haml

```diff
@@ -0,0 +1,22 @@
+%h1 Listing clubs
+
+%table
+  %thead
+    %tr
+      %th Name
+      %th Description
+      %th Logo
+      %th
+
+  %tbody
+    - @clubs.each do |club|
+      %tr
+        %td= link_to club.name, club_path(club)
+        %td= link_to club.description, club_path(club)
+        %td= image_tag club.logo, size: '100x150'
+        %td
+          - if current_user
+            = link_to 'Join', join_club_path(club)
```

Comment:
Sure. That's the way we imagined it from the beggining : - )

On 6 May 2015 at 17:14, vistulahub <notifications@github.com> wrote:

> In app/views/clubs/index.html.haml
> <https://github.com/ror2015/vistula/pull/6#discussion_r29767795>:
>
> > +  %thead
> > +    %tr
> > +      %th Name
> > +      %th Description
> > +      %th Logo
> > +      %th
> > +
> > +  %tbody
> > +    - @clubs.each do |club|
> > +      %tr
> > +        %td= link_to club.name, club_path(club)
> > +        %td= link_to club.description, club_path(club)
> > +        %td= image_tag club.logo, size: '100x150'
> > +        %td
> > +          - if current_user
> > +            = link_to 'Join', join_club_path(club)
>
> what if a user already joined to the club? please check events pull
> request, there's a solution for that
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ror2015/vistula/pull/6/files#r29767795>.
>